www/docs/client/configuration.mdx

```diff
@@ -142,6 +142,8 @@ new DrupalClient(process.env.NEXT_PUBLIC_DRUPAL_BASE_URL, {
 })
 ```
 
+You can find more info about using a custom cache [here](/docs/client/cache).
```

Comment:
Ah nice catch. I did read this twice or more but somehow did not see this. Thank you.